pkg/trace/agent/agent.go

```diff
@@ -586,37 +613,33 @@ func (a *Agent) getAnalyzedEvents(pt *traceutil.ProcessedTrace, ts *info.TagStat
 	return events
 }
 
-// runSamplers runs all the agent's samplers on pt and returns the sampling decision
-// along with the sampling rate.
+// runSamplers runs all the agent's samplers on pt and returns the sampling decision along with the
+// sampling rate.
+//
+// The rare sampler is run first, catching all rare traces early. If the Probabilistic sampler is
```

Comment:
Need to update this doc now that the Probabilistic Sampler is invoked elsewhere.